pkg/loki/loki.go

```diff
@@ -360,12 +363,13 @@ func (t *Loki) setupModuleManager() error {
 	deps := map[string][]string{
 		Ring:            {RuntimeConfig, Server, MemberlistKV},
 		Overrides:       {RuntimeConfig},
-		Distributor:     {Ring, Server, Overrides},
+		TenantConfigs:   {RuntimeConfig},
+		Distributor:     {Ring, Server, Overrides, TenantConfigs},
 		Store:           {Overrides},
-		Ingester:        {Store, Server, MemberlistKV},
-		Querier:         {Store, Ring, Server, IngesterQuerier},
-		QueryFrontend:   {Server, Overrides},
-		Ruler:           {Ring, Server, Store, RulerStorage, IngesterQuerier, Overrides},
+		Ingester:        {Store, Server, MemberlistKV, TenantConfigs},
```

Comment:
Not sure about adding it here, but you're future proofing the modules in case we add to the tenant configs